app/views/support/providers/schools_check/show.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= form_with(model: @school_form, url: support_recruitment_cycle_provider_check_school_path(@provider.recruitment_cycle_year, @provider), method: :put, local: true) do |f| %>
 
     <fieldset class="govuk-fieldset">
-      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l"> <span class="govuk-caption-l"><%= t("support.providers.schools.new") %> - <%= @provider.provider_name %> </span>
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l"> <span class="govuk-caption-l"><%= t("support.providers.schools.new") %> - <%= "#{@provider.provider_name} (#{@provider.provider_code})" %> </span>
```

Comment:
I think there is a helper for this https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/decorators/provider_decorator.rb#L22

Reply:
Not sure it is in scope here ```>> provider.name_and_code
!! #<NameError: undefined local variable or method `provider' for #<ActionView::Base:0x00000000606558>
Did you mean?  provide
               @provider>
>> @provider.name_and_code
!! #<NoMethodError: undefined method `name_and_code' for #<Provider id: 18396,```

Reply:
`provider.decorate.name_and_code?`

Reply:
If you put `decorates_assigned :provider` in the schools_check_controller.rb, it still doesn't work
```
[1] pry(#<#<Class:0x0000000115431938>>)> provider.name_and_code
NoMethodError: undefined method `name_and_code' for #<Provider id: 18396
```
am I missing something?

Reply:
I think it is because I am calling `def provider` already

Reply:
I would have thought `provider.decorate.name_and_code` would work? Otherwise `ProviderDecorator.new(provider)` in your `def provider`